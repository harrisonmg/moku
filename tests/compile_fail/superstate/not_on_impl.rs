mod blinky {
    use moku::*;

    struct Top {}

    #[superstate(Top)]
    struct Bottom {}
}

fn main() {}
