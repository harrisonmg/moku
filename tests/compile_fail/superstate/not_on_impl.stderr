error: expected `impl`
 --> tests/compile_fail/superstate/not_on_impl.rs:7:5
  |
7 |     struct Bottom {}
  |     ^^^^^^
